easystepper/easystepper.go

```diff
@@ -49,6 +79,68 @@ func (d *Device) Move(steps int32) {
 	d.stepNumber = int32(stepN)
 }
 
+// Off turns off all motor pins
+func (d *Device) Off() {
+	d.pins[0].Low()
+	d.pins[1].Low()
+	d.pins[2].Low()
+	d.pins[3].Low()
+}
+
+// Move rotates the motors the number of given steps
+// (negative steps will rotate it the opposite direction)
+func (d *DualDevice) Move(stepsA, stepsB int32) {
```

Comment:
Have not gone thru the details, but seems like this method could be cleaned up further...